TextStateFeature can't be used in client components

### Describe the Bug

The app crashes when importing defaultColors into my JSX converter and using the "use client" directive.

This prevents the usage of the rich text component in things like forms, FAQ drop downs, and other client component use cases.

The error received is:
```
⨯ ./node_modules/.pnpm/pino-abstract-transport@2.0.0/node_modules/pino-abstract-transport/index.js:6:34
Module not found: Can't resolve 'worker_threads'
  4 | const split = require('split2')
  5 | const { Duplex } = require('stream')
> 6 | const { parentPort, workerData } = require('worker_threads')
    |                                  ^
  7 |
  8 | function createDeferred () {
  9 |   let resolve

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./node_modules/.pnpm/pino-pretty@13.0.0/node_modules/pino-pretty/index.js
./node_modules/.pnpm/payload@3.40.0_graphql@16.11.0_typescript@5.7.2/node_modules/payload/dist/utilities/logger.js
./node_modules/.pnpm/payload@3.40.0_graphql@16.11.0_typescript@5.7.2/node_modules/payload/dist/index.js
./node_modules/.pnpm/@payloadcms+richtext-lexical@3.40.0_@faceless-ui+modal@3.0.0-beta.2_react-dom@19.0.0_re_9f0f42bf4404651e339d4e63c2e7b4be/node_modules/@payloadcms/richtext-lexical/dist/index.js
./src/components/RichText/converters/textConverter.tsx
./src/components/RichText/converters/index.tsx
./src/components/RichText/index.tsx
./src/blocks/ClientBlock/Component.tsx
```

### Link to the code that reproduces this issue

https://github.com/nlvcodes/test-repo

### Reproduction Steps

1. Clone repo
2. Initialize first user
3. Create a post with a `ClientBlock` block (already created with "use client"). Include highlighting.
4. Publish content.
7. The app crashes with the error message provided above. (this likely happens before you even create a user)
8. Comment out "use client" from the component, and the TextStateFeature works as intended.

### Which area(s) are affected? (Select all that apply)

plugin: richtext-lexical

### Environment Info

```text
Binaries:
  Node: 23.6.0
  npm: 10.9.2
  Yarn: 3.6.1
  pnpm: 10.6.2
Relevant Packages:
  payload: 3.40.0
  next: 15.3.3
  @payloadcms/db-mongodb: 3.40.0
  @payloadcms/next/utilities: 3.40.0
  @payloadcms/payload-cloud: 3.40.0
  @payloadcms/richtext-lexical: 3.40.0
  react: 19.0.0
  react-dom: 19.0.0
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 24.5.0: Tue Apr 22 19:53:27 PDT 2025; root:xnu-11417.121.6~2/RELEASE_ARM64_T6041
  Available memory (MB): 49152
  Available CPU cores: 14
```

--- a/packages/richtext-lexical/src/exports/client/index.ts
+++ b/packages/richtext-lexical/src/exports/client/index.ts
@@ -32,6 +32,7 @@ export { LexicalPluginToLexicalFeatureClient } from '../../features/migrations/l
 export { SlateToLexicalFeatureClient } from '../../features/migrations/slateToLexical/feature.client.js'
 export { ParagraphFeatureClient } from '../../features/paragraph/client/index.js'
 export { DebugJsxConverterFeatureClient } from '../../features/debug/jsxConverter/client/index.js'
+export { defaultColors } from '../../features/textState/defaultColors.js'
 
 export { RelationshipFeatureClient } from '../../features/relationship/client/index.js'
 

